tests/test_inherits_path_option.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import os
+
+from twitter.common.contextutil import environment_as, temporary_dir
+
+from pex.pex_builder import PEXBuilder
+from pex.testing import run_simple_pex, write_simple_pex
+
+
+def write_inheriting_pex(td, exe_contents):
+  """Write a pex file that contains an executable entry point
+
+  :param td: temporary directory path
+  :param exe_contents: entry point python file
+  :type exe_contents: string
+  :param dists: distributions to include, typically sdists or bdists
+  :param coverage: include coverage header
+  """
+  with open(os.path.join(td, 'exe.py'), 'w') as fp:
+    fp.write(exe_contents)
+
+  pb = PEXBuilder(path=td, preamble=None)
+  pb.info.inherit_path = True
+  pb.set_executable(os.path.join(td, 'exe.py'))
+  pb.freeze()
+
+  return pb
+
+
+def test_inherits_path_option():
+  with temporary_dir() as td:
+    pb = write_inheriting_pex(td, 'import sys\nimport os\nprint(os.environ)\nprint(sys.path)')
```

Comment:
This is a bit deceiving (here and below in `test_does_not_inherit_path_option`) since the environ and sys.path printing goes unused.  You might just use an empty script to emphasize the point of the test is checking the verbose output.